src/etc/rc.filter_synchronize

```diff
@@ -266,10 +266,16 @@ if (is_array($config['hasync'])) {
 		}
 		$sections[] = 'wol';
 	}
-	if ($hasync['synchronizetrafficshaper'] != "" and is_array($config['shaper'])) {
+	if ($hasync['synchronizetrafficshaper'] != "") {
+		if (!is_array($config['shaper'])) {
```

Comment:
You can safely do `init_config_arr(array('shaper'))` and skip the manual check. So, you could just do this:
```
if ($hasync['synchronizetrafficshaper'] != "") {
	init_config_arr(array('shaper'));
	$sections[] = 'shaper';
}
```

Reply:
ok, done